test/library/standard/Map/testToArray.chpl

```diff
@@ -1,6 +1,8 @@
 use Map;
 
-var m = new map(int, int);
+config param parSafe = false;
+
+var m = new map(int, int, parSafe);
 for i in -10..10 do
   m[i] = -i;
```

Comment:
Can you change this call to `m.add(i, -i)` instead of `m[i] = -i`. Using `this()` while `parSafe=true` now produces a compiler warning.